client/dashboard/dashboard-charts/config.js

```diff
@@ -9,16 +14,23 @@ import { charts as revenueCharts } from 'analytics/report/revenue/config';
 import { charts as categoriesCharts } from 'analytics/report/categories/config';
 import { charts as couponsCharts } from 'analytics/report/coupons/config';
 import { charts as taxesCharts } from 'analytics/report/taxes/config';
+import { charts as downloadsCharts } from 'analytics/report/downloads/config';
+
+const DASHBOARD_CHARTS_FILTER = 'woocommerce_admin_dashboard_charts_filter';
 
-const allCharts = ordersCharts
-	.map( d => ( { ...d, endpoint: 'orders' } ) )
-	.concat(
-		productsCharts.map( d => ( { ...d, endpoint: 'products' } ) ),
-		revenueCharts.map( d => ( { ...d, endpoint: 'revenue' } ) ),
-		categoriesCharts.map( d => ( { ...d, endpoint: 'categories' } ) ),
-		couponsCharts.map( d => ( { ...d, endpoint: 'orders' } ) ),
-		taxesCharts.map( d => ( { ...d, endpoint: 'taxes' } ) )
-	);
+const allCharts = applyFilters(
+	DASHBOARD_CHARTS_FILTER,
+	ordersCharts
+		.map( d => ( { ...d, endpoint: 'orders' } ) )
+		.concat(
+			productsCharts.map( d => ( { ...d, endpoint: 'products' } ) ),
+			revenueCharts.map( d => ( { ...d, endpoint: 'revenue' } ) ),
```

Comment:
Not related to this PR, but just saw it now. The _Revenue_ report is the first report in the menu but here we list its charts as it was the third one, which makes Dashboard charts order incoherent with reports order.

In #690 there seems to be the order we should use for Dashboard charts, but we are not following it. Should we open an issue to tackle that and get the input from @LevinMedia?

Reply:
@Aljullu 

>Should we open an issue to tackle that

Yes please 👍 And thank you!

Reply:
Yeah, that makes sense. I'll just address that here before merging.

Reply:
I did a basic re-order for this one because I can easily move groups, but opened a new issue to look at titles and order in depth, https://github.com/woocommerce/woocommerce-admin/issues/2266